src/main/scala/xiangshan/backend/MemBlock.scala

```diff
@@ -81,7 +81,8 @@ class ooo_to_mem(implicit p: Parameters) extends MemBlockBundle {
    val pendingPtrNext = Input(new RobPtr)
   }
 
-  val isStore = Input(Bool())
+  val isStore = Input(Bool()) // TODO: re-name the signal
+  val isVls = Input(Bool()) // TODO: re-name the signal
```

Comment:
I also think renaming these two signals should be better.

Reply:
DONE